policy.go

```diff
@@ -86,6 +94,7 @@ type pdpServer struct {
 // requests and replies using PDP server.
 type policyPlugin struct {
 	endpoints       []string
+	rule            policyRule
```

Comment:
we expect : rules []PolicyRule
it is an array and the mechanism is to loop in order over this array until having a firm reply.